pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -584,6 +606,27 @@ func (tc *TidbCluster) PDStsDesiredOrdinals(excludeFailover bool) sets.Int32 {
 	return GetPodOrdinalsFromReplicasAndDeleteSlots(replicas, tc.getDeleteSlots(label.PDLabelVal))
 }
 
+func (tc *TidbCluster) PDMSStsDesiredReplicas(componentName string) int32 {
+	if tc.Spec.PDMS == nil {
+		return 0
+	}
+
+	for _, component := range tc.Spec.PDMS {
```

Comment:
no auto failover is supported now, right? How about adding some comments or TODOs?

Reply:
add TODO